balancer-js/src/modules/zaps/bbausd2-migrations/bbausd1.integration.spec.ts

```diff
@@ -0,0 +1,256 @@
+import dotenv from 'dotenv';
+import { expect } from 'chai';
+import hardhat from 'hardhat';
+import {
+  BalancerError,
+  BalancerErrorCode,
+  Network,
+  RelayerAuthorization,
+  PoolModel,
+  Subgraph,
+  SubgraphPoolRepository,
+} from '@/.';
+import { BigNumber } from '@ethersproject/bignumber';
+import { Contracts } from '@/modules/contracts/contracts.module';
+import { ADDRESSES } from './addresses';
+import { BbaUsd1Builder as MigrationBuilder } from './bbausd1';
+import { setBalance } from '@nomicfoundation/hardhat-network-helpers';
+import { parseEther, formatEther } from '@ethersproject/units';
+import { JsonRpcSigner } from '@ethersproject/providers';
+import { MaxUint256 } from '@ethersproject/constants';
+
+import { Interface } from '@ethersproject/abi';
+import { PoolsProvider } from '@/modules/pools/provider';
+const liquidityGaugeAbi = ['function deposit(uint value) payable'];
+const liquidityGauge = new Interface(liquidityGaugeAbi);
+
+/*
+ * Testing on GOERLI
```

Comment:
REMINDER - Update these tests to mainnet versions once live?

Reply:
mainnet vs goerli should be only different in addresses configuration, so just passing a different chainId should be enought